pipeline_svc/conflict_manager.go

```diff
@@ -369,14 +369,18 @@ func (c *ConflictManager) formatTargetDoc(input *crMeta.ConflictParams) *mc.MCRe
 
 	bodylen := 0
 	var spec SubdocMutationPathSpec
-	specs := make([]SubdocMutationPathSpec, 0, 5)
+	specs := make([]SubdocMutationPathSpec, 0, 6)
```

Comment:
The hard-coded 6 here is not all used.
If mv is nil or if pv is nil, we're allocating but not using all 6.

Can we have a dynamic variable and check if pv mv is nil and allocate as appropriate?
This will also add context to the reason for allocating X number, without needing to read every code below

Reply:
Done